src/back2top.less

```diff
@@ -0,0 +1,20 @@
+.to-top {
+    opacity: 1;
+    display: inline-block;
+    padding: 1em;
+    position: fixed;
+    bottom: 1em;
+    right: 1em;
+    .paper-btn {
+    padding: 0.6em 1em;
```

Comment:
could these lines be indented to show that its within the paper-btn class

Reply:
Done